### ISSUE TITLE
Spacing between rows of an aligned environment too tight

### ISSUE THREAD

**Issue by arnog** [Khan/KaTeX#686]
The gap between rows of an array is smaller than what TeX is producing.

**KaTeX**
<img width="306" alt="screen shot 2017-04-27 at 4 46 22 pm" src="https://cloud.githubusercontent.com/assets/760798/25508764/32893cd0-2b69-11e7-98f2-aac35644514e.png">
**LaTeX**
<img width="208" alt="screen shot 2017-04-27 at 4 46 29 pm" src="https://cloud.githubusercontent.com/assets/760798/25508763/327278ec-2b69-11e7-85b9-9296ba455359.png">

This might be because line 602 of buildHTML.js should be
`        let gap = arstrutDepth;
`
instead of
`        let gap = 0;
`

**Comment by edemaine** [comment]
@arnog Sounds like a good find.  Could you include the LaTeX source for testing?

**Comment by kevinbarabash** [comment]
Here's the code:
```
        let gap = 0;
        if (group.value.rowGaps[r]) {
            gap = calculateSize(group.value.rowGaps[r].value, style);
            if (gap > 0) { // \@argarraycr
                gap += arstrutDepth;
                if (depth < gap) {
                    depth = gap; // \@xargarraycr
                }
                gap = 0;
            }
        }

        outrow.height = height;
        outrow.depth = depth;
        totalHeight += height;
        outrow.pos = totalHeight;
        totalHeight += depth + gap; // \@yargarraycr
        body[r] = outrow;
```

I'm not sure what precise behavior is supposed to be here.

**Comment by kevinbarabash** [comment]
Also reported in https://github.com/Khan/KaTeX/issues/531 and https://github.com/Khan/KaTeX/issues/312.

**Comment by arnog** [comment]
Sorry, I should have been more explicit. Here's a simpler case demonstrating the issue.

`\begin{aligned}A\\B\end{aligned}`
**TeX:**
<img width="48" alt="screen shot 2017-05-01 at 10 40 06 pm" src="https://cloud.githubusercontent.com/assets/760798/25605356/2d2327a2-2ebf-11e7-9ff2-f6d2a606b13e.png">
**KaTeX**
<img width="59" alt="screen shot 2017-05-01 at 10 40 00 pm" src="https://cloud.githubusercontent.com/assets/760798/25605357/2d24b07c-2ebf-11e7-88b9-8789645e82cb.png">

The problem is specific to the `aligned` environment. It turns out that because of the equation number, there's an extra gap of 3pt (by default) inserted in between the lines of an aligned environment. It's called '\jot' in the LaTeX source.

**Comment by edemaine** [comment]
Fixed in #725
